tests/Tests/Unit/Common/Forms/FormVitalsTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * FormVitalsTest.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+/**
+ * FormVitalsTest.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Tests\Unit\Common\Forms;
+
+use OpenEMR\Common\Database\QueryUtils;
+use OpenEMR\Common\Forms\FormVitalDetails;
+use OpenEMR\Common\Forms\FormVitals;
+use OpenEMR\Services\ListService;
+use PHPUnit\Framework\TestCase;
+
+class FormVitalsTest extends TestCase
+{
+    const NOTE_UNIT_TEST = "OEUnitTest";
+    const VITAL_DETAILS_COLUMN = "bps";
+
+    protected function tearDown(): void
+    {
+        QueryUtils::sqlStatementThrowException("DELETE FROM " . FormVitalDetails::TABLE_NAME
+            . " WHERE form_id IN (select id FROM " . \OpenEMR\Common\Forms\FormVitals::TABLE_NAME . " WHERE note=?)", [self::NOTE_UNIT_TEST]);
```

Comment:
shouldn't need full `\OpenEMR\Common\Forms\FormVitals` for this call and others below

Reply:
Should be fixed.